ci/.gitlab-ci.yml

```diff
@@ -1,442 +1,101 @@
 
 ##
 ## RUNS
 ##
 
-.run_common:
-  stage: test
-  trigger:
-    strategy: depend
-
-cpu release test gcc11:
-  extends: .run_common
-  needs:
-    - cpu release build gcc11
-  trigger:
-    include:
-      - artifact: pipeline.yml
-        job: cpu release build gcc11
-
-cpu release test gcc11 stdexec:
-  extends: .run_common
-  needs:
-    - cpu release build gcc11 stdexec
-  trigger:
-    include:
-      - artifact: pipeline.yml
-        job: cpu release build gcc11 stdexec
-
-cpu debug test gcc11 stdexec:
-  extends: .run_common
-  needs:
-    - cpu debug build gcc11 stdexec
-  trigger:
-    include:
-      - artifact: pipeline.yml
-        job: cpu debug build gcc11 stdexec
-
-cpu release test gcc12 cxx20:
-  extends: .run_common
-  needs:
-    - cpu release build gcc12 cxx20
-  trigger:
-    include:
-      - artifact: pipeline.yml
-        job: cpu release build gcc12 cxx20
-
-cpu release test clang12:
-  extends: .run_common
-  needs:
-    - cpu release build clang12
-  trigger:
-    include:
-      - artifact: pipeline.yml
-        job: cpu release build clang12
-
-cpu release test clang13 cxx20:
-  extends: .run_common
-  needs:
-    - cpu release build clang13 cxx20
-  trigger:
-    include:
-      - artifact: pipeline.yml
-        job: cpu release build clang13 cxx20
-
-cpu codecov test gcc11:
-  extends: .run_common
-  needs:
-    - cpu codecov build gcc11
-  trigger:
-    include:
-      - artifact: pipeline.yml
-        job: cpu codecov build gcc11
-
-cuda release test gcc11:
-  extends: .run_common
-  needs:
-    - cuda release build gcc11
-  trigger:
-    include:
-      - artifact: pipeline.yml
-        job: cuda release build gcc11
-
-cuda release test gcc11 scalapack:
-  extends: .run_common
-  needs:
-    - cuda release build gcc11 scalapack
-  trigger:
-    include:
-      - artifact: pipeline.yml
-        job: cuda release build gcc11 scalapack
-
-cuda codecov test gcc11:
-  extends: .run_common
-  needs:
-    - cuda codecov build gcc11
-  trigger:
-    include:
-      - artifact: pipeline.yml
-        job: cuda codecov build gcc11
+#cuda release test gcc11:
+#  extends: .run_common
+#  needs:
+#    - cuda release build gcc11
+#  trigger:
+#    include:
+#      - artifact: pipeline.yml
+#        job: cuda release build gcc11
+#
+#cuda release test gcc11 scalapack:
+#  extends: .run_common
+#  needs:
+#    - cuda release build gcc11 scalapack
+#  trigger:
+#    include:
+#      - artifact: pipeline.yml
+#        job: cuda release build gcc11 scalapack
+#
+#cuda codecov test gcc11:
+#  extends: .run_common
+#  needs:
+#    - cuda codecov build gcc11
+#  trigger:
+#    include:
+#      - artifact: pipeline.yml
+#        job: cuda codecov build gcc11
+#
+#cuda codecov test gcc11 scalapack:
+#  extends: .run_common
+#  needs:
+#    - cuda codecov build gcc11 scalapack
+#  trigger:
+#    include:
+#      - artifact: pipeline.yml
+#        job: cuda codecov build gcc11 scalapack
 
```

Comment:
Here you have too many empty lines at the end of the file.

Reply:
Fixed